libr/include/r_debug.h

```diff
@@ -91,7 +91,6 @@ typedef enum {
 	R_DEBUG_REASON_DEAD = -1,
 	R_DEBUG_REASON_NONE = 0,
 	R_DEBUG_REASON_SIGNAL,
-	R_DEBUG_REASON_SEGFAULT,
```

Comment:
dont remove, segfault is quite generic stop reason concept, not just tied to signals